src/icons/index.ts

```diff
@@ -1,4 +1,6 @@
 import * as faBars from '@fortawesome/free-solid-svg-icons/faBars';
+import * as faMoneyBill from '@fortawesome/free-solid-svg-icons/faMoneyBill';
+import * as faTachometerAlt from '@fortawesome/free-solid-svg-icons/faTachometerAlt';
```

Comment:
These should actually be part of `@patternfly/react-icons` now. Upgrading should give these two icons and the bars icon now too.

Reply:
Thanks!